src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +48,45 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        try {
+            return Optional.of(
```

Comment:
it's good, but use less lines, please :)

Reply:
fixed